### PR TITLE
EG-1197 Synchronize choice stanza with viewer for date updates

### PR DESCRIPTION
**Pull request by woollett-r**

### CODE REVIEW

File: app/core/models/ocelot/stanzas/ChoiceStanza.scala [hunk 1]
```diff
@@ -67,46 +70,61 @@ sealed trait ChoiceTest {
   val right: String
   def eval(labels: Labels): Boolean
   def value(arg: String, labels: Labels): String = labelReference(arg).fold(arg)(ref => labels.value(ref).getOrElse(""))
-  def op(f: (BigDecimal, BigDecimal) => Boolean, g: (String, String) => Boolean, labels: Labels): Boolean = {
+  def moreThanOrEquals(date: LocalDate, other: LocalDate): Boolean = date.compareTo(other) >= 0
```
Comment (cmrhmrc) [inline app/core/models/ocelot/stanzas/ChoiceStanza.scala:73]: See comment on viewer PR

File: test/core/models/ocelot/stanzas/ChoiceStanzaSpec.scala [hunk 2]
```diff
@@ -144,7 +144,7 @@ class ChoiceStanzaSpec extends BaseSpec {
 
   "ChoiceStanza" must {
 
-    "deserialise from json" in {
+    "deserialize from json" in {
```
Comment (cmrhmrc) [inline test/core/models/ocelot/stanzas/ChoiceStanzaSpec.scala:147]: As viewer PR

**Review by cmrhmrc — CHANGES_REQUESTED** [review]
ChoiceTest fns as comment

File: test/core/models/ocelot/stanzas/ChoiceStanzaSpec.scala (hunk 2 above)
Comment (woollett-r) [reply test/core/models/ocelot/stanzas/ChoiceStanzaSpec.scala:146]: Done

**Pull request merged**